src/main/java/com/codessquad/qna/service/UserService.java

```diff
@@ -41,6 +41,16 @@ public User findUserByUserId(String userId) {
     }
 
     public User findById(Long id) {
-        return userRepository.findUserById(id);
+        return userRepository.findById(id).get();
```

Comment:
`Optional`은 `get()`으로 꺼내지 말아주세요!!
http://homoefficio.github.io/2019/10/03/Java-Optional-%EB%B0%94%EB%A5%B4%EA%B2%8C-%EC%93%B0%EA%B8%B0/
요기 참고해주세요!

Reply:
> If a value is present, returns the value, otherwise throws NoSuchElementException.

JavaDoc엔 이렇게 적혀있습니다.
보통 `get()`을 사용할 때에는, `ifPresent()`를 사용해주세요.

Reply:
변경했습니다!